modules/ROOT/nav.adoc

```diff
@@ -21,6 +21,7 @@
  ** xref:proxy-deploy-runtime-fabric.adoc[Deploy an API Proxy to Anypoint Runtime Fabric]
 * xref:upgrade-index.adoc[Upgrade Runtime Fabric]
  ** xref:upgrade-cluster.adoc[Upgrade Cluster]
+ ** xref:upgrade-14path.adoc[Upgrade Path to Runtime Fabric Version 1.4]
```

Comment:
I think we can generalize this for any Runtime Fabric component upgrade. Based upon the wording used in "upgrade-index.adoc", we could use the term "components".

"Upgrade components"

Reply:
This can be in both v1.3 / v1.4

Reply:
I made some updates - thank you!